crates/core/c8y_smartrest/src/operations.rs

```diff
@@ -101,10 +128,16 @@ impl Operations {
     }
 }
 
-fn get_operations(dir: impl AsRef<Path>, cloud_name: &str) -> Result<Operations, OperationsError> {
+fn get_operations(
+    dir: impl AsRef<Path>,
+    cloud_name: Option<&str>,
```

Comment:
By making that `cloud_name` optional, the meaning of this function has become a bit convoluted. I feel the simplest thing would be to just take the `dir` parameter and include the `cloud_name` in the calling function itself.